lib/src/lib/table/cell-editors/cell-select-editor/cell-select-editor.component.ts

```diff
@@ -66,21 +60,33 @@ export class CellSelectEditorComponent extends CellEditorAbstract<CellSelectEdit
         return o.options;
       }
       return [];
-    })
+    }),
+    distinctUntilChanged()
   );
 
-  @ViewChild(MatSelect)
-  private matSelect?: MatSelect;
+  private _matSelect?: MatSelect;
 
-  public readonly defaultCompareFn = (a: any, b: any): boolean => a === b;
+  @ViewChild(MatSelect)
+  private set matSelect(value: MatSelect) {
+    this._matSelect = value;
+    setTimeout(() => {
+      if (this._matSelect && !this._matSelect.panelOpen) {
+        this._matSelect.focus();
+        this._matSelect.open();
```

Comment:
doesn't this mean that all mat selects will open automatically on initialisation of the component?

Reply:
This was already the case but done differently